book/src/rust-client/counter_contract_tutorial.md

```diff
@@ -0,0 +1,678 @@
+# Deploying a Counter Contract
+*Using the Miden client in Rust to deploy and interact with a custom smart contract on Miden*
+
+## Overview
+In this tutorial, we will build a simple counter smart contract that maintains a count, deploy it to the Miden testnet, and interact with it by incrementing the count.
```

Comment:
Can you say that in this tutorial we use our deployed testnet node, but a user can also test against a local node